docs/spec/generics.rst

```diff
@@ -93,8 +93,9 @@ This is equivalent to omitting the generic notation and just saying
 User-defined generic types
 --------------------------
 
-You can include a ``Generic`` base class to define a user-defined class
-as generic.  Example::
+You can define a user-defined class as generic by including a ``Generic``
+base class, either explicitly or implicitly via the ``Protocol[T]``
```

Comment:
Maybe we can change the `Protocol[T]` part a bit here? Why `T`?

Reply:
This sentence also sounds like it predates PEP 695, so it may be worth rephrasing a bit more.

Reply:
Thanks, I rephrased it. PTAL :)